database-bridge-lambda/src/sql/Item.ts

```diff
@@ -186,3 +187,21 @@ export const getItemCounts = (
     WHERE "pinboardId" IN ${sql(args.pinboardIds)} 
     GROUP BY "pinboardId"
 `;
+
+export const getUniqueUsersPerHourInRange = async (
+  sql: Sql,
+  range: Range
+): Promise<[number, number][]> => {
+  const result = await sql`
+        SELECT DATE_TRUNC('hour', "timestamp") as "hour",
+               COUNT(DISTINCT "userEmail")     as "uniqueUsers"
+        FROM "Item"
+        WHERE "timestamp" >= ${range.from}
+          AND "timestamp" < ${range.to}
+        GROUP BY "hour"
+    `;
+  return Array.from(result.values()).map((row) => [
+    parseInt(row.uniqueUsers),
+    row.hour.getTime(),
+  ]);
+};
```

Comment:
you can map a promise without having an await using `.then` (see others in this file for examples)
```suggestion
): Promise<[number, number][]> => 
  sql`
        SELECT DATE_TRUNC('hour', "timestamp") as "hour",
               COUNT(DISTINCT "userEmail")     as "uniqueUsers"
        FROM "Item"
        WHERE "timestamp" >= ${range.from}
          AND "timestamp" < ${range.to}
        GROUP BY "hour"
    `.then(rows => rows.map((row) => [
    parseInt(row.uniqueUsers),
    row.hour.getTime(),
  ]));
```